xtask-wasm-run-example/src/lib.rs

```diff
@@ -2,6 +2,39 @@ use quote::{quote, quote_spanned};
 use syn::spanned::Spanned;
 use syn::{parse, parse_macro_input};
 
+/// This macro helps to run an example in the project's `examples/` directory using a development
+/// server.
+///
+/// # Usage
+///
+/// * In the file `examples/my_example.rs`, create your example:
+///
+///     ```rust,ignore
+///     use wasm_bindgen::prelude::*;
+///
+///     #[wasm_bindgen]
+///     extern "C" {
+///         #[wasm_bindgen(js_namespace = console)]
+///         fn log(message: &str);
+///     }
+///
+///     #[xtask_wasm::run_example]
+///     fn run_app() {
+///         log::("Hello World!");
+///     }
+///     ```
+///
+/// * In the project's `Cargo.toml`:
+///
+///     ```toml
+///     [dev-dependencies]
+///     xtask-wasm = { version = "*", features = ["run-example"] }
+///     ```
+/// * Then to run the development server with the example:
```

Comment:
```suggestion
///     ```
///
/// * Then to run the development server with the example:
```